src/webview/css/static/diff2html-tweaks.css

```diff
@@ -17,11 +17,6 @@ ins {
   z-index: 1;
 }
 
-/* workaround until https://github.com/rtfpessoa/diff2html/pull/445 is merged */
```

Comment:
We still need this workaround because the fix hasn't been released yet on diff2html side.

Reply:
Oh, I missed the fact that we bundle diff2html now; I've updated to the fixed version 3.4.19.

Reply:
Great! The fix has been released on NPM (though the last one on GitHub was in 2021 😅).